tgui/packages/tgui/styles/interfaces/PINInputModal.scss

```diff
@@ -0,0 +1,47 @@
+/**
+ * @file
+ * @copyright 2024
+ * @author ZeWaka (https://github.com/ZeWaka)
+ * @license MIT
+ */
+
+@import "../colors.scss";
+
+.PINInput {
+  display: flex;
+  justify-content: center;
+
+  .Button {
+    display: flex;
+    justify-content: center;
+    width: 3rem;
+    height: 3rem;
```

Comment:
Rather than hardcoding these, does throwing `grow` on the relevant `Stack.Item`s work instead?